specification/security/resource-manager/readme.md

```diff
@@ -77,7 +77,7 @@ csharp:
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.Security
   payload-flattening-threshold: 2
-  output-folder: $(csharp-sdks-folder)/Security/Management.Security/Generated
+  output-folder: $(csharp-sdks-folder)/SecurityCenter/Management.SecurityCenter/Generated

```

Comment:
- Is the product being renamed to Microsoft.SecurityCenter from Microsoft.Security?
- Is the name of the nuget package also being changed?
  - If so then what happens to the old package? Is it being deprecated?

Your responses to the above questions will help us decide if similar things need to be done for python, js, go and ruby sdks.

Reply:
No, it's [already called Microsoft.SecurityCenter](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SecurityCenter/Management.SecurityCenter/Generated/AutoProvisioningSettingsOperations.cs) (see path)
Nuget should not be affected by this change.